src/androidTest/kotlin/LogcatTest.kt

```diff
@@ -1,13 +1,14 @@
 package logcat
 
-import com.google.common.truth.Truth.assertThat
 import logcat.LogPriority.INFO
-import org.junit.After
-import org.junit.Test
+import kotlin.test.AfterTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
 
 class LogcatTest {
 
-  @After
+  @AfterTest
```

Comment:
Interesting, why migrate from junit to kotlin test? Is kotlin test able to run on multiple platforms?

Reply:
Exactly.  In fact, I discovered that the @kotlin.test.AfterTest is just a typealias to @org.junit.After for JVM

```
package kotlin.test

public actual typealias Test = org.junit.Test
public actual typealias Ignore = org.junit.Ignore
public actual typealias BeforeTest = org.junit.Before
public actual typealias AfterTest = org.junit.After
```

But using that allowed me to port the tests to JS and Native (which was super helpful in figuring out the bit that were different between the Android implementation and other platforms, especially with stack traces, Companion objects and frozen references for native, to fill in the `expect`/`actual`.  Running the Gradle `check` target executes each of the platform's tests.